spotirec.py

```diff
@@ -233,17 +236,17 @@ def check_tune_validity(tune: str):
     :param tune: tune input as string
     """
     if not tune.split('_', 1)[0] in tune_prefix:
-        print(f'Tune prefix \"{tune.split("_", 1)[0]}\" is malformed - available prefixes:')
-        print(tune_prefix)
+        logging.warning(f'Tune prefix \"{tune.split("_", 1)[0]}\" is malformed - available prefixes:')
+        logging.warning(tune_prefix)
         exit(1)
     if not tune.split('=')[0].split('_', 1)[1] in tune_attr:
-        print(f'Tune attribute \"{tune.split("=")[0].split("_", 1)[1]}\" is malformed - available attributes:')
-        print(tune_attr)
+        logging.warning(f'Tune attribute \"{tune.split("=")[0].split("_", 1)[1]}\" is malformed - available attributes:')
+        logging.warning(tune_attr)
         exit(1)
     try:
         float(tune.split('=')[1]) if '.' in tune.split('=')[1] else int(tune.split('=')[1])
     except ValueError:
-        print(f'Tune value {tune.split("=")[1]} is not a valid integer or float value')
+        logging.warning(f'Tune value {tune.split("=")[1]} is not a valid integer or float value')
         exit(1)
```

Comment:
Maybe these should be errors, actually. Either that or this function should return a boolean - in this case we could just skip the tune on line 716 if it isn't valid. Not sure about this though, as the user might want to re-do the input rather than just creating a playlist without the tune.